scripts/fetch/dv_data.R

```diff
@@ -12,7 +12,7 @@ fetch.dv_data <- function(viz){
   sites <- deps[["sites"]]
   
   startDate <- paste0(year, "-01-01")
-  endDate <- paste0(year, "-12-31")
+  endDate <- paste0(year, "-01-02")
```

Comment:
Safe to revert this change now?

Reply:
Yes, although really I'd say these dates should all be moved into the viz.yaml so they can be set in a single place

Reply:
Fair. Up to you how you do it, but note that we're not currently sure that anybody will be developing this interactive viz any more this year, so code quality/maintainability matters somewhat less than usual.

Reply:
Right, probably wouldn't bother doing it now